bin/setenv-docker-customize.sh

```diff
@@ -299,7 +299,7 @@ else
 
   if [ "${EXO_PROXY_SSL}" = "true" ]; then
     xmlstarlet ed -L -s "/Server/Service/Connector" -t attr -n "scheme" -v "https" \
-      -s "/Server/Service/Connector" -t attr -n "secure" -v "false" \
```

Comment:
Due last issues with SAML, this changes should be checked in such context before being deployed

Reply:
This is another context, the reported issue was due to the `SameSite=Strict` cookies restriction. I'll make some checks anyway. Thanks!